build.sbt

```diff
@@ -11,7 +11,9 @@ resolvers ++= {
   else Seq.empty
 }
 
-libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0-SNAPSHOT"
+libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0"
+
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.12")
```

Comment:
That said, as noted in sbt gitter, I said I would do it this way until it became an issue . ie now :smile: So, I'll have a quick look to get a working version that we can refine later